### PR TITLE
bowl is open

### LINKED ISSUE #20452: Bowl needs to be opened before pouring liquid
## Description

Bowls need to be "opened" before pouring something into them.

**Reproduction**

- Grab an empty bowl
- Try to fill it with liquid/flour/rice
- Click/'Z' on bowl and then you can pour liquid into it

**Screenshots**

Bowl

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
slightly unfucked bowl and ingredient packet parenting
bowl no longer has to be opened, fixes #20452
also every ingredient bag/pack/carton can be smashed now not just flour

- ReagentContainerBase
- - FoodBowlBig
- - ReagentPacketBase
- - - every bag/pack/carton for ingredients

- ItemHeftyBase
- - flour bag/pack
- - eggs tomatoes watermelons

## Why / Balance
good

## Technical details
no

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
no cl no fun

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml [hunk 1]
```diff
@@ -26,22 +28,59 @@
...
       types:
         Blunt: 0
-  - type: TrashOnEmpty
-    solution: food
 
 - type: entity
-  parent: ReagentContainerBase
+  abstract: true
+  id: ItemHeftyBase
+  description: hit hard ye
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:71]: Yeet this and the rest is guzzy

**Review by AJCM-git — DISMISSED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:71]: but 4/5 things use it

Comment (AJCM-git) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:71]: I mean the desc, not the prot

Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:71]: bang stick gibber [DEBUG] reference fake

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**